sql/schema.sql

```diff
@@ -15,7 +15,9 @@ DROP TABLE IF EXISTS `ghtorrent`.`users` ;
 CREATE TABLE IF NOT EXISTS `ghtorrent`.`users` (
   `id` INT(11) NOT NULL AUTO_INCREMENT COMMENT '',
   `login` VARCHAR(255) NOT NULL COMMENT '',
+  `name` VARCHAR(255) NULL DEFAULT NULL COMMENT '',
```

Comment:
The schema is automatically constructed the first time GHTorrent connects to the database. See the migration code under `lib/ghtorrent/migrations`